### PR TITLE
Add support for Gilded Tools

### LINKED ISSUE #11146: Gilded Axe is not supported by Woodcutting plugin
**Describe the bug**
When woodcutting the Plugin always stays as "NOT woodcutting"

**To Reproduce**
1. Click on a tree while using gilded axe
2. See woodcutting plugin update logs cut but not status

**Expected behavior**
It should change from red "NOT woodcutting" text to green "woodcutting"

**Screenshots**
https://i.imgur.com/dQac8dk.png

**Environment (please complete the following information):**
 - OS: Windows 10
 - RuneLite version: 1.6.10.1
 - Launcher version: 2.1.0

### PR DESCRIPTION
**Pull request by Hydrox6**
Closes #11146

Add support for the Gilded Axe and its animation to woodcutting, idle notifier, and wintertodt.

Add support for the Gilded Pickaxe in the Idle Notifier.

### CODE REVIEW

**Comment by makingcomment** [comment]
Does support for the gilded pickaxe need to be added as well?

**Comment by Hydrox6** [comment]
Good point, though I don't have a Gilded pick to find that information (ironman btw). If you have one and want to help, @ me on the Discord please.

**Comment by Nightfirecat** [comment]
This should also add the MLM pickaxe animation

File: runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOverlay.java [hunk 1]
```diff
@@ -58,7 +59,7 @@
 	private static final Set<Integer> MINING_ANIMATION_IDS = ImmutableSet.of(
 		MINING_MOTHERLODE_BRONZE, MINING_MOTHERLODE_IRON, MINING_MOTHERLODE_STEEL,
 		MINING_MOTHERLODE_BLACK, MINING_MOTHERLODE_MITHRIL, MINING_MOTHERLODE_ADAMANT,
-		MINING_MOTHERLODE_RUNE, MINING_MOTHERLODE_DRAGON, MINING_MOTHERLODE_DRAGON_UPGRADED,
+		MINING_MOTHERLODE_RUNE, MINING_MOTHERLODE_GILDED, MINING_MOTHERLODE_DRAGON, MINING_MOTHERLODE_DRAGON_UPGRADED,
 		MINING_MOTHERLODE_DRAGON_OR, MINING_MOTHERLODE_INFERNAL, MINING_MOTHERLODE_CRYSTAL
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOverlay.java]: Keep these 3 to a line like before.
```suggestion
		MINING_MOTHERLODE_RUNE, MINING_MOTHERLODE_GILDED, MINING_MOTHERLODE_DRAGON,
		MINING_MOTHERLODE_DRAGON_UPGRADED, MINING_MOTHERLODE_DRAGON_OR, MINING_MOTHERLODE_INFERNAL,
		MINING_MOTHERLODE_CRYSTAL
```

Comment (Hydrox6) [reply runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeOverlay.java]: Wow I didn't even notice that

**Pull request merged**